frame/session/src/lib.rs

```diff
@@ -170,6 +170,10 @@ impl<
 			offset
 		})
 	}
+
+	fn weight(_now: BlockNumber) -> Weight {
+		0
+	}
```

Comment:
Can you document/comment why this is zero? That's surprising to me.

Reply:
This has no storage reads, and trivial computation.

Reply:
i will add a doc comment here, and also point out it may want to be re-evaluated